pwem/protocols/protocol_import/images.py

```diff
@@ -94,7 +94,7 @@ def _acquisitionWizardCondition(self):
 
     # --------------------------- INSERT functions ----------------------------
     def _insertAllSteps(self):
-
+        self._fillFileNamesList()
```

Comment:
This mfails in come tests.... I think those that overwrite _insertAllSteps and therefore does not call _fiilFileNameList

 scipion3 tests pwem.tests.protocols.test_protocols_sets.TestSets